### PR TITLE
Light and dark theme headerbar button border improvements

### PR DESCRIPTION
**Pull request by Feichtmeier**
- unify the color of the :checked button borders in the headerbar, because uneven border colors make the button look more pressed than necessary without improving the legibility
- this is also more consistent with the default theme :checked headerbar buttons

Before:
image
image

After:
image
image

Default theme:
image

@madsrh @clobrano @ubuntujaggers ?

### CODE REVIEW

**Comment by Feichtmeier** [comment]
Peek 2020-02-11 20-05
Peek 2020-02-11 20-04

**Comment by clobrano** [comment]
It's not so clear :D, can you rephrase it? The title as well

**Comment by Feichtmeier** [comment]
Done - hope this is more clear? :D

**Comment by clobrano** [comment]
Yes thanks. I think I prefer the press effect in light/dark master though 😖

**Comment by Feichtmeier** [comment]
Why exactly? It's less visible in master and adds a 3d effect. There is no benefit and it looks inconsistent with the rest of the theme

**Comment by clobrano** [comment]
Because the press effect is 3D by definition. Basically the PR replace the
press with a stronger border

**Comment by Feichtmeier** [comment]
The press effect is pressing the leveled up button down. 
It's also less visible. This was the reason we've changed the default theme btw. (Which also has no second layer 3d effect)

**Comment by clobrano** [comment]
I know, but now that I see the difference closely, I prefer the 3D effect.
It's just my opinion though, if the others prefer this PR I have
no problems :)

**Comment by ubuntujaggers** [comment]
Looking at the gifs I slightly prefer the change - but possibly because my eyes don't interpret the uneven border as a 3D effect, it just looks like one of the edges being darker (almost like a slight mistake rather than a design choice).

**Comment by clobrano** [comment]
Alright then :), feel free to merge it +1
